embedding-calculator/src/docs/find_faces_post.yml

```diff
@@ -4,16 +4,16 @@ summary: 'Find faces in the given image and return their bounding boxes.'
 description: 'Returns bounding boxes of detected faces on the image.'
 operationId: findFacesPost
 consumes:
-  - multipart/form-data
+  - application/json
```

Comment:
Please, revert it, it should be `multipart/form-data`

Reply:
Fixed